scipy/signal/spectral.py

```diff
@@ -511,11 +509,401 @@ def spectrogram(x, fs=1.0, window=('tukey',.25), nperseg=256, noverlap=None,
     if noverlap is None:
         noverlap = nperseg // 8
 
-    freqs, time, Pxy = _spectral_helper(x, x, fs, window, nperseg, noverlap,
+    freqs, time, Sxx = _spectral_helper(x, x, fs, window, nperseg, noverlap,
                                         nfft, detrend, return_onesided, scaling,
-                                        axis, mode=mode)
+                                        axis, mode='psd')
 
-    return freqs, time, Pxy
+    return freqs, time, Sxx
+
+
+def check_COLA(window, nperseg, noverlap, tol=1e-10):
```

Comment:
Is this meant to be public? If so it needs to be added to `__all__` and the docs. If not, then it should be prefixed by `_`